### PR TITLE
Update developer pop documentation to reference minikube 0.32.0. Late…

### PR DESCRIPTION
**Pull request by glennslaven**
…st version of minikube tested.

### CODE REVIEW

File: content/tutorials/developer-workflow/setting-up-your-developer-pop.md [hunk 1]
```diff
@@ -10,13 +10,10 @@ Section.io's Developer PoP runs on Kubernetes. Minikube is a prepackaged Kuberne
 Minikube runs on your preference of virtual machine ([VirtualBox] is a free option).
 
 1. Verify that you have installed VirtualBox or another VM software.
-1. Download and install <a href="https://github.com/kubernetes/minikube/releases/tag/v0.26.1" target="Minikube" title="Minikube v0.26.1 download">Minikube v0.26.1</a>.
+1. Download and install <a href="https://github.com/kubernetes/minikube/releases/tag/v0.32.0" target="Minikube" title="Minikube v0.32.0 download">Minikube v0.32.0</a>.
```
Comment (jstangroome) [inline content/tutorials/developer-workflow/setting-up-your-developer-pop.md]: I assumed we'd add 0.32.0 to the list, not revoke our published support for 0.26.1

Comment (glennslaven) [reply content/tutorials/developer-workflow/setting-up-your-developer-pop.md]: We don't really have a list, is there a reason the keep the older versions?

Comment (glennslaven) [reply content/tutorials/developer-workflow/setting-up-your-developer-pop.md]: Having only the later versions means we don't need to have the two different commands for setting up devpop

Comment (glennslaven) [reply content/tutorials/developer-workflow/setting-up-your-developer-pop.md]: https://www.section.io/docs/tutorials/developer-workflow/setting-up-your-developer-pop/

**Pull request merged**